package.json

```diff
@@ -16,16 +16,29 @@
     "changelog": "uphold-scripts changelog $npm_package_version",
     "lint": "uphold-scripts lint",
     "release": "uphold-scripts release",
-    "test": "uphold-scripts test",
+    "test": "TZ=UTC jest",
     "version": "uphold-scripts version"
   },
+  "jest": {
+    "collectCoverage": true,
+    "coveragePathIgnorePatterns": ["/node_modules/"],
+    "modulePaths": ["<rootDir>"],
+    "restoreMocks": true,
+    "testRegex": "test/.*\\.test.js$"
```

Comment:
Do we confirmed that this will cover all the files?

Reply:
Typo, "did we confirm"

Reply:
☝️ by me :trollface: